### ISSUE TITLE
fan Control can't seem to ¨find¨ my T_Sensor

### ISSUE THREAD

**Issue by TheNewNoobie** [Rem0o/FanControl.Releases#1500]
Hi, I wanted to know if it is something I'm missing or is not yet supported. I can see my T_Sensor in other programs as I use an water loop and see the coolants temp, but fan control cant seem to see it as I want the fans to respond to the loops coolant temp instead of the cup/gpu temp as they can be violent. So my question is if it's something I'm missing or just not in yet.

Specs:
Mb: Asus strix maximus z790 hero
Connected T_Sensor: 1/4 guage temp sensor from corsair

**Comment by Rem0o** [comment]
Do you have the asus ec checkbox enabled in sensor settings?

**Comment by PHaumering** [comment]
I have the same problem with the ASUS rog strix b450i and its T_Sensor. It shows up correctly in e.g. CPUID HWMonitor, but not in FanControl. I can confirm that "Embedded EC (Asus)" is checked in the sensor settings.

**Comment by Rem0o** [comment]
The EC entry probably needs to be added in for your specific motherboard.

See https://github.com/Rem0o/FanControl.Releases#issues-and-hardware-compatibility

**Comment by andino66** [comment]
I'm having the same issue. I have a x670e Hero. 

It would be nice if the Water Out T Sensor would show up.

**Comment by reidprichard** [comment]
I'm having the same issue with a case temperature sensor on my Asus ROG B350-F Strix. Shows up in HWiNFO64 as T_Sensor. I also have "Embedded EC (Asus)" checked in settings. 

I've tried adding a few plugins to see if I can get to the sensor that way, but I can't get it working for some reason. I suppose that's a separate issue.

**Comment by p4nchez** [comment]
same for me on a ASUS Z390 F Gaming
have an water temp sensor called T_Sensor1 pluggit in to motherboard sensor head
Bios and HWiNFO can show it but under FanControl Sensor dont appear even with Asus plugin :(

so i cant used this nice app because i want my fans to use water temp from my loop

**Comment by bombaysaph** [comment]
I have the same issue on an Asus Strix Z790-F Gaming WiFi

**Comment by oozkaya** [comment]
I don't know if it can help but here is what worked for me !

On my side I am using HWINFO to monitor my temps and I'm looking at the "Water In T Sensor" for my watercooling. 

So I just started to use this plugin https://github.com/Rem0o/FanControl.HWInfo and I can now see my sensor in FanControl.

**Comment by bombaysaph** [comment]
yes thank you, I ran somewhere into this plugin solution with hwinfo. that works for me as well, would be a nice qol update to include this sensor by default without running hwinfo in parallel
